src/event/invoice_payment.rs

```diff
@@ -156,7 +164,10 @@ fn hackmarket_purchase<'a>(
                         price,
                         format!("the {} you tried to buy has is no longer on the market", name),
                     ),
-                    dm_blocks(paid_invoice.invoicee.clone(), vec![json!({
+                    dm_blocks(
+                        paid_invoice.invoicee.clone(),
+                        "Sale failed! Your GP has been refunded.".to_string(),
```

Comment:
This is actually the failure to purchase, not the failure to sell. Including more information here may also be valuable, although isn't necessary.

Reply:
as mentioned [here](https://github.com/hackagotchi/hackagotchi/pull/66#discussion_r447856294) - I'm just trying to give a basic idea of what happened, so that users will open the app.

I'll update to "purchase failed," got those flipped around somehow.